### ISSUE TITLE
Develop a USP

### ISSUE THREAD

**Issue by vegantriathlete** [LearningDevOps/intro-to-devops#1]
Akshat emailed:

I looked at the curriculum. I think we don't have the USP in yet. We need at least one topic which makes us better than others. I hope you understand what I mean.

My motivation of adding the example project was to differentiate it from "only theory" courses but I understand that it will be time-taking. Any other ideas to create that USP?

**Comment by vegantriathlete** [comment]
Akshat,
That's a fair point about creating a USP. It's worth thinking about. However, I do not believe that all Udemy courses have a USP. I believe that there are many successful Udemy courses without a USP. People take the courses because they are marketed well and / or they think the instructor does a better job teaching it.

While Udacity's course is a good course, I think that there are some definite benefits to Udemy's platform compared to Udacity's. And I think that the Udacity course is most likely our biggest competition. I'm not going to worry too much about competing with them, though. I think that Udemy draws an absolutely HUGE crowd and we can do fine even if we don't capture the students who will choose to use Udacity.

So, sure we can think more about creating a USP. But, I don't think that needs to hold us up from creating and publishing the course. Let's get it out there and see what kind of response we can get. We can always continue to improve it. I'm more interested in getting the MVP out the door as fast as possible. If we can come up with a good USP prior to launch, that will be wonderful. If we can't, it's not a show-stopper IMHO.

Does what I am saying make sense to you?

I think it would make an absolutely fantastic Blab for us to talk about this more. Would you like to schedule one for Wednesday? We can do it at 8AM Mountain / 7:30 PM IST.

**Comment by akshatgoel** [comment]
I agree with the point of going through the MVP first. Let's go over the curriculum once.

We can work on USP over iterations. Let's focus on great content and value for MVP.

**Comment by vegantriathlete** [comment]
Sounds great! Thanks for tagging the issue and setting up the milestone.
